app/controllers/posts_controller.rb

```diff
@@ -4,21 +4,11 @@ class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
     @posts = @user.posts.includes(:comments)
```

Comment:
- [ ] Just as the previous reviewer suggested, there seems to be an error when creating a post as a result non verification of CSRF token authenticity as you can see in the attached image. 
![image](https://github.com/Zeph180/blogit/assets/46839548/c035bd15-c038-4308-9721-2406dc380f9f)

It was suggested that you add the line of code found below in you `PostsController` 👇 
```ruby
skip_before_action :verify_authenticity_token, only: [:destroy, :create]
```
I will also advise that you remove the below code found on line 3 in your `PostsController`, otherwise the error will persist
```ruby
protect_from_forgery prepend: true
```